src/main/java/racingcar/entry/Car.java

```diff
@@ -0,0 +1,29 @@
+package racingcar.entry;
+
+import java.util.Random;
+
+public class Car {
+
+    private final Random random;
```

Comment:
지금은 Random을 받아도 괜찮은데
요구사항이 변경되어 다른 환경에서는 범위가 넓은 랜덤값 또는 외부 요인(날씨, API)으로 전진이 판단된다면 Car 도메인이 많이 변경될 것 같아요
전략 패턴을 사용해서 리팩토링 해보면 어떨까요?

```java
interface MovePolicy {
    boolean isMovable();
}
```

```suggestion
    private final MovePolicy policy;
```